### PR TITLE
[OYPD-332] Configure permissions for Editor and Contributor

### PR DESCRIPTION
**Pull request by hamrant**
Jira - https://propeople-us.atlassian.net/browse/OYPD-332
d.org issue - https://www.drupal.org/node/2858426

**Steps for review:**
- [x] login as admin
- [x] go to /admin/people/permissions
- [x] check that editor and contributor has minimum permissions
- [x] go to /admin/people/create
- [x] create user with contributor role
- [x] login as contributor
- [x] try to add any content
- [x] profit

### CODE REVIEW

**Comment by gundevel** [comment]
Build comment file: 
--------------
===============

--------------
Test FAILed.
Refer to this link for build results (access rights to CI server needed): 
http://openy-dev.ffwua.com:8080/job/PR_BUILDER/684/

**Build Log**
*last 25 lines*

 ```
[...truncated 593 lines...]
Do you wish to run all pending updates? (y/n): y

FATAL: all hosts have already failed -- aborting

PLAY RECAP ******************************************************************** 
           to retry, use: --limit @/var/lib/jenkins/reinstall.retry

localhost                  : ok=20   changed=12   unreachable=0    failed=1   

FATAL: command execution failed
hudson.AbortException: Ansible playbook execution failed
	at org.jenkinsci.plugins.ansible.AnsiblePlaybookBuilder.perform(AnsiblePlaybookBuilder.java:227)
	at org.jenkinsci.plugins.ansible.AnsiblePlaybookBuilder.perform(AnsiblePlaybookBuilder.java:200)
	at hudson.tasks.BuildStepCompatibilityLayer.perform(BuildStepCompatibilityLayer.java:75)
	at hudson.tasks.BuildStepMonitor$1.perform(BuildStepMonitor.java:20)
	at hudson.model.AbstractBuild$AbstractBuildExecution.perform(AbstractBuild.java:779)
	at hudson.model.Build$BuildExecution.build(Build.java:205)
	at hudson.model.Build$BuildExecution.doRun(Build.java:162)
	at hudson.model.AbstractBuild$AbstractBuildExecution.run(AbstractBuild.java:537)
	at hudson.model.Run.execute(Run.java:1741)
	at hudson.model.FreeStyleBuild.run(FreeStyleBuild.java:43)
	at hudson.model.ResourceController.execute(ResourceController.java:98)
	at hudson.model.Executor.run(Executor.java:381)
ERROR: Ansible playbook execution failed
```

**Comment by hamrant** [comment]
test this please

**Comment by gundevel** [comment]
Build comment file: 
--------------
===============
 
| Environment | Link |
| ------------- | ------------- |
| Fresh OpenY installation | http://openy-dev.ffwua.com/build685 |
| Upgraded(upgrade path) installation | http://upgrade.openy-dev.ffwua.com/build685 |
| Installation process | http://install.openy-dev.ffwua.com/build685/install.php |

--------------
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://openy-dev.ffwua.com:8080/job/PR_BUILDER/685/

**Comment by gundevel** [comment]
Behat test results file URL: http://openy-dev.ffwua.com/build685/profiles/openy/build/reports/behat

File: modules/openy_features/openy_user/openy_user.module [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+ * @file
+ * Contains openy_user.module.
+ */
+
+use Drupal\Core\Entity\EntityInterface;
+
+/**
+ * Implements hook_node_presave().
+ */
+function openy_user_node_presave(EntityInterface $node) {
```
Comment (alexschedrov) [inline modules/openy_features/openy_user/openy_user.module]: Let's allow contributor to publish. It's not so required right now.

**Comment by alexschedrov** [comment]
1. Contributor and Editor have no access to WYSIWYG
image

2. Contributor and Editor are not able to select image in Media borwser

**Comment by gundevel** [comment]
Build comment file: 
--------------
===============
 
| Environment | Link |
| ------------- | ------------- |
| Fresh OpenY installation | http://openy-dev.ffwua.com/build691 |
| Upgraded(upgrade path) installation | http://upgrade.openy-dev.ffwua.com/build691 |
| Installation process | http://install.openy-dev.ffwua.com/build691/install.php |

--------------
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://openy-dev.ffwua.com:8080/job/PR_BUILDER/691/

**Comment by gundevel** [comment]
Behat test results file URL: http://openy-dev.ffwua.com/build691/profiles/openy/build/reports/behat

**Comment by bjwilkins** [comment]
Build comment file: 
--------------
===============
 
| Environment | Link |
| ------------- | ------------- |
| Fresh OpenY installation | http://openy-dev.ffwua.com/build694 |
| Upgraded(upgrade path) installation | http://upgrade.openy-dev.ffwua.com/build694 |
| Installation process | http://install.openy-dev.ffwua.com/build694/install.php |

--------------
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://openy-dev.ffwua.com:8080/job/PR_BUILDER/694/

**Comment by bjwilkins** [comment]
Behat test results file URL: http://openy-dev.ffwua.com/build694/profiles/openy/build/reports/behat

**Review by alexschedrov — APPROVED** [review]
Approved.

**Pull request merged**